Configuration/Settings.yaml

```diff
@@ -44,6 +44,7 @@ Neos:
       overrideHostname: ''
       analytics:
         id: false
+        anonymizeIp: false
```

Comment:
When defining it instead like 
```
parameters:
  anonymize_ip: true
```

You could support any number of parameters

Reply:
Done

Reply:
I meant when you define the parameter here with the actual name and just use the config to render the json for Fusion, you can simplify the Fusion part. Now the naming doesn't match anymore.

Reply:
Ah got it.